samples/07.advanced-web-chat-apps/b.sso-for-enterprise/azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+pool:
```

Comment:
The .yml files were marked by git as deleted (and the new one added), as opposed to renamed. I'm assuming this is because there was a significant enough change to the file to consider it a new file.